app/assets/stylesheets/application.scss

```diff
@@ -9,13 +9,31 @@ $primary-color-dark: false !default;
 @if not $primary-color-dark {
   $primary-color-dark: darken($primary-color, 15%);
 }
-$secondary-color: color("teal", "lighten-1") !default;
+$secondary-color: color("amber", "darken-1") !default;
 $success-color: color("green", "base") !default;
 $error-color: color("red", "base") !default;
-
 $link-color: color("light-blue", "darken-1") !default;
 
+/*** Additional variables ***/
+$background-color: #FFF;
+$brand-font-color: color("brown", "base");
+$primary-font-color: color("brown", "darken-4");
+
+$button-color-raised: $primary-font-color !default;
+
+/*** Body ***/
+body { background-color: #E6E0CF; }
```

Comment:
Color literals like `#E6E0CF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E6E0CF` should be written as `#e6e0cf`